### PR TITLE
YQ-2068 remove unused IFunctionRegistry from compute actors

### PR DESCRIPTION
**Pull request by zverevgeny**
### Changelog category 

* Not for changelog (changelog entry is not required)

### CODE REVIEW

File: ydb/library/yql/dq/actors/compute/dq_compute_actor_impl.h [hunk 1]
```diff
@@ -1244,7 +1242,7 @@ class TDqComputeActorBase : public NActors::TActorBootstrapped<TDerived>
         IRandomProvider* randomProvider
         )
     {
-        auto collectStatsLevel = StatsModeToCollectStatsLevel(RuntimeSettings.StatsMode);
+                auto collectStatsLevel = StatsModeToCollectStatsLevel(RuntimeSettings.StatsMode);
```
Comment (rvu1024) [inline ydb/library/yql/dq/actors/compute/dq_compute_actor_impl.h:1245]: style

File: ydb/library/yql/dq/actors/compute/dq_compute_actor_impl.h [hunk 2]
```diff
@@ -1461,12 +1459,12 @@ class TDqComputeActorBase : public NActors::TActorBootstrapped<TDerived>
 
 private:
     void InitializeTask() {
-        for (ui32 i = 0; i < Task.InputsSize(); ++i) {
+                for (ui32 i = 0; i < Task.InputsSize(); ++i) {
```
Comment (rvu1024) [inline ydb/library/yql/dq/actors/compute/dq_compute_actor_impl.h:1462]: style

File: ydb/library/yql/dq/actors/compute/dq_compute_actor_impl.h [hunk 3]
```diff
@@ -1461,12 +1459,12 @@ class TDqComputeActorBase : public NActors::TActorBootstrapped<TDerived>
...
             const auto& inputDesc = Task.GetInputs(i);
             Y_ABORT_UNLESS(!inputDesc.HasSource() || inputDesc.ChannelsSize() == 0); // HasSource => no channels
 
             if (inputDesc.HasTransform()) {
-                auto result = InputTransformsMap.emplace(
+                                auto result = InputTransformsMap.emplace(
```
Comment (rvu1024) [inline ydb/library/yql/dq/actors/compute/dq_compute_actor_impl.h:1467]: style

**Review by rvu1024 — CHANGES_REQUESTED** [review]

File: ydb/library/yql/dq/actors/compute/dq_compute_actor_impl.h (hunk 2 above)
Comment (zverevgeny) [reply ydb/library/yql/dq/actors/compute/dq_compute_actor_impl.h]: fixed

**Review by gridnevvvit — APPROVED** [review]

**Review by rvu1024 — APPROVED** [review]

**Pull request merged**